src/components/Items/ProfileContact.js

```diff
@@ -14,46 +18,75 @@ const ProfileContact = ({
   showArrow,
   userId,
   item,
-  ImageComponent = null
-}) => (
-  <Pressable
-    onPress={onPress}
-    android_ripple={{
-      color: COLORS.gray1,
-      borderless: false,
-      borderRadius: 10
-    }}
-    style={styles.pressable}>
-    <View style={styles.container}>
-      <View style={styles.profile}>
-        {ImageComponent || (
-          <Image
-            testID="image"
-            style={styles.image}
-            source={{uri: photo !== '' ? photo : undefined}}
-          />
-        )}
-        <Text style={styles.fullname}>{fullname}</Text>
-      </View>
-      {showArrow && (
-        <>
-          {userId !== item.user_id && (
-            <View>
-              <Mi name="arrow-forward-ios" size={18} />
-            </View>
+  ImageComponent = null,
+  disabled = false,
+  from
+}) => {
+  const handleYouText = () => {
+    if (!from && (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER)) {
+      return '(You)';
+    }
+    return '';
+  };
+
+  return (
+    <Pressable
+      onPress={onPress}
+      android_ripple={{
+        color: COLORS.gray1,
+        borderless: false,
+        borderRadius: 10
+      }}
+      disabled={!showArrow || disabled}
+      style={styles.pressable}>
+      <View style={styles.container}>
+        <View style={styles.profile}>
+          {ImageComponent || (
+            <Image
+              testID="image"
+              style={styles.image}
+              source={{uri: photo !== '' ? photo : undefined}}
+            />
           )}
-        </>
-      )}
-      {select && (
-        <View testID="selected">
-          <MemoIc_Checklist />
+          <Text testID="name" style={styles.fullname(userId === item?.user_id)}>
+            {fullname} {handleYouText()}
+          </Text>
         </View>
-      )}
-    </View>
-  </Pressable>
-);
+        {showArrow && (
+          <>
+            {userId !== item.user_id && (
+              <View>
+                <Mi name="arrow-forward-ios" size={18} />
+              </View>
+            )}
+          </>
+        )}
+        {select && (
+          <View testID="selected">
+            <MemoIc_Checklist />
+          </View>
+        )}
+      </View>
+    </Pressable>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Pressable` component has been updated to include a `disabled` prop which disables the component based on the `showArrow` and `disabled` props. The `ImageComponent` is now rendered conditionally and the `fullname` text now includes the result of the `handleYouText` function. The `userId` comparison has been moved inside the `showArrow` condition. These changes seem to be necessary for the new functionality and there doesn't appear to be any issues with them.



<!-- This is an auto-generated comment by OSS CodeRabbit -->